.github/workflows/pytest-gpu.yml

```diff
@@ -75,11 +81,42 @@ jobs:
 
     - name: Test examples with MPI
       run: |
-        DEVITO_MPI=1 mpirun -n 2 pytest examples/seismic/acoustic/acoustic_example.py
-        DEVITO_MPI=1 mpirun -n 2 pytest examples/seismic/elastic/elastic_example.py
-        DEVITO_MPI=1 mpirun -n 2 pytest examples/seismic/tti/tti_example.py
-        DEVITO_MPI=1 mpirun -n 2 pytest examples/seismic/viscoacoustic/viscoacoustic_example.py
-        DEVITO_MPI=1 mpirun -n 2 pytest examples/seismic/viscoelastic/viscoelastic_example.py
+        # Tests currently being executed individually due to issue #1536
+        source ~/environments/${{ matrix.language }}/bin/activate
+        source ~/environments/scripts/${{ matrix.language }}_env.sh
+        DEVITO_MPI=1 mpirun -n 2 pytest examples/seismic/acoustic/acoustic_example.py::test_isoacoustic_stability[OT2-1]
```

Comment:
Really needs each one to be called separately?

Reply:
The underlying issue is a memory leak with OpenMP, I'll see if I can track it down